File: examples/swagger-2/codegen-sdk/swagger2/typescript/axios/src/model/paging/Pageable.ts
/* tslint:disable */
/* eslint-disable */

        import {Sort} from "./Sort";
import {DefaultOrderField} from "feign-client";



export interface  Pageable {

        paged: boolean;
        unpaged: boolean;
        offset: number;
        pageSize: number;
        pageNumber: number;
        sort: Sort;
}

File: examples/swagger-2/src/test/resources/asserts/java/retrofit/src/com/wuxp/codegen/swagger2/model/paging/Pageable.java
package com.wuxp.codegen.swagger2.model.paging;

import lombok.Data;
import lombok.experimental.Accessors;
import jakarta.validation.constraints.*;
import com.fasterxml.jackson.annotation.JsonFormat;
        import com.wuxp.codegen.swagger2.model.paging.Sort;

@Data
@Accessors(chain = true)
public class  Pageable {

        private String paged;

        private String unpaged;

        private Integer offset;

        private Integer page_size;

        private Integer page_number;

        private Sort sort;

}
